Samples_for_Extending_Pentaho/Reference Implementations/Security/SAML 2.0/pentaho-saml/pom.xml

```diff
@@ -177,7 +177,7 @@
     <dependency>
       <groupId>org.springframework.security</groupId>
       <artifactId>spring-security-web</artifactId>
-      <version>${dependency.spring.security.version}</version>
+      <version>${spring-security.version}</version>
```

Comment:
Isn't this dep on the parent dep management?

Reply:
yes, agree